app/models/service_template_ansible_playbook.rb

```diff
@@ -157,6 +155,7 @@ def update_catalog_item(options, auth_user = nil)
   end
 
   def destroy
```

Comment:
When run this method at `after_detroy` `resource_actions` are already gone. We may save references to job_templates at `before_destroy` callback.